### PR TITLE
Fixing update

### PR DESCRIPTION
**Pull request by lucie-pl**
All CRUD method working now.

### CODE REVIEW

File: app/controllers/recipes_controller.rb [hunk 1]
```diff
@@ -1,5 +1,8 @@
 class RecipesController < ApplicationController
-  before_action :set_recipe, only: [:show, :edit, :update, :destroy]
+  before_action :set_recipe, only: [:show, :update, :destroy]
+
+  # before_update :mark_recipe_ingredient_for_destruction
```
Comment (quetzai) [inline app/controllers/recipes_controller.rb]: remove commented code and extra lines

File: app/views/recipes/_edit_recipe_ingredient_fields.html.erb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      <%= f.label :unity, "Unité" %>
+      <%= f.text_field :unity %>
+    </div>
+
+    <%= cocooned_remove_item_link 'Supprimer l\'ingrédient', f, method: :delete %>
+
+    <%# <%= link_to "Supprimer l'ingrédient'",
+    recipe_recipe_ingredient_path(@recipe_ingredient),
+    data: {turbo_method: :delete, turbo_confirm: "Es-tu sûr.e ?"}
+    %>
```
Comment (quetzai) [inline app/views/recipes/_edit_recipe_ingredient_fields.html.erb]: is this commented? let's remove it then

**Review by quetzai — COMMENTED** [review]
tiny things, the rest is OK

File: app/controllers/recipes_controller.rb (hunk 1 above)
Comment (lucie-pl) [reply app/controllers/recipes_controller.rb]: DONE

File: app/views/recipes/_edit_recipe_ingredient_fields.html.erb (hunk 2 above)
Comment (lucie-pl) [reply app/views/recipes/_edit_recipe_ingredient_fields.html.erb]: DONE

**Pull request merged**